open-api/rest-catalog-open-api.yaml

```
@@ -3265,6 +3265,133 @@ components:
           additionalProperties:
             type: string
 
+    ReadRestrictions:
+      type: object
+      description: >
+          Read restrictions for a table, including column projections and row filter expressions, according to the current schema.
```

Need to update this since expressions are going to use ID references.
```suggestion
          Read restrictions for a table, including column projections and row filter expressions.
```